### ISSUE TITLE
Multiple Tables Within Tabs

### ISSUE THREAD

**Issue by AlexanderAdelAU** [Jo-Geek/jQuery-ResizableColumns#2]
If there are multiple tables on tabs the plugin appears to work only on the first table.  The second table's column widths get minimised and no dragging works.

**Comment by Jo-Geek** [comment]
Yes currently the plugin only applies on the first table it finds. Will update to work on all the tables.
